index.html

```diff
@@ -107,7 +107,7 @@ <h2>Tonic</h2>
               /></a>
             </li>
           </ul>
-          <button class="project-button">See Project</button>
+          <button class="project-button projButtOne">See Project</button>
```

Comment:
Hi @BenMukebo. I'm a little confused about having to redo the "works" section because the info from the objects is different in the template. Do I need to remove that section from the HTML file and create new objects for the normal section of the cards in javascript?

Reply:
Yeah